gulpfile.js

```diff
@@ -92,7 +107,7 @@ var frontendDependencies = manifest.dependencies.frontend;
 var cssFilesPath = [];
 var jsFilesPath = [];
 var fontFolderPath = [];
-var cssBackgroundPath = [];
+var cssBackgroundFilepaths = [];
 var generatedTargetDir = path.join(
   'third_party', 'generated',
   isMinificationNeeded ? 'prod' : 'dev');
```

Comment:
I think I am going to make that a separate task. And since this is useful  during build then i Will make it to be dependent task for build task.